packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -32,6 +32,7 @@ const TABLE = 'mog-prs';
 const datastore = new Datastore();
 const MAX_TEST_TIME = 1000 * 60 * 60 * 6; // 6 hr.
 const WORKER_SIZE = 4;
+const MAX_ENTRIES = 100;
```

Comment:
I think `100` is potentially enough entries that it would start to hit some API limits, I would be tempted to pick a threshold like `25`.